pkg/platform/kube/test/platform_test.go

```diff
@@ -52,6 +58,87 @@ type DeployFunctionTestSuite struct {
 	KubeTestSuite
 }
 
+func (suite *DeployFunctionTestSuite) TestDeployFromGit() {
```

Comment:
This integration test doesnt need Kubernetes platform and is time consuming to CI
let's leave Nuclio <> Kubernetes parts here.

This test should leave under `pkd/processor/build/test/build_test.go` and should include some scenarios
- git:
  - clone from repo / branch / no auth
  - clone from repo / tag / no auth
  - clone from repo with basic auth (PAT or username+password)
 
 For the latter, you can skip tests by default, but please do make sure
1. working with azure devops / github / bitbucket
2. working with branches / tags